site/src/app/about/page.tsx

```diff
@@ -1,17 +1,266 @@
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[3]
+        : theme.colors.gray[5],
+  },
+
+  name: {
+    fontFamily: `Greycliff CF, ${theme.fontFamily}`,
+  },
+}));
+
+interface UserInfoActionProps {
+  avatar: string;
+  name: string;
+  email: string;
+  job: string;
+}
+
+// Create an array of dev information objects
+const devs = [
+  {
+    avatar: "/avatar1.webp",
+    title: "Full Stack Developer, DevOps, IT Ops",
+    name: "David Gama",
+    linkedin: "https://www.linkedin.com/in/gamadavid/",
+    github: "https://github.com/gamadjg",
+    portfolio: "https://github.com/gamadjg",
+  },
+  {
+    avatar: "/avatar2.webp",
+    title: "Programer & reader",
+    name: "Sabih S.",
+    linkedin: "https://www.linkedin.com/in/sabihsarowar/",
+    github: "https://github.com/kleenkanteen",
+    portfolio: "https://www.sabih.me/",
+  },
+  {
+    avatar: "/avatar4.webp",
+    title: "Software Engineer",
+    name: "Kenny Tram",
+    linkedin: "https://www.linkedin.com/in/kennytram/",
+    github: "https://github.com/kennytram",
+    portfolio: "https://kennytram.github.io/",
+  },
+  {
+    avatar: "/avatar3.webp",
+    title: "Full Stack Web dev & Software Engineer",
+    name: "Lucile Tronczyk",
+    linkedin: "https://www.linkedin.com/in/lucile-tronczyk/",
+    github: "https://github.com/LucileTech",
+    portfolio: "https://lucile-tech.com/",
+  },
+  {
+    avatar: "/avatar5.webp",
+    title: "Software Engineer",
+    name: "Joyce Yu",
+    linkedin: "https://www.linkedin.com/in/joyceyu-sf/",
+    github: "https://github.com/joyceycodes",
+    portfolio: "https://joyceyu.netlify.app/",
+  },
+];
+
+export default function About() {
+  const { classes } = useStyles();
 
-export default function Completed() {
   return (
     <>
-      <main className="flex min-h-screen flex-col items-center justify-between p-24">
-        <div className="z-10 max-w-5xl w-full items-center justify-between font-mono text-sm lg:flex">
-          <AppName />
-        </div>
-        <div className="relative flex place-items-center ">
-          <h1>About us</h1>
+      <main className="flex flex-col min-h-screen p-4">
+        {/* Header */}
+        <HeaderAction />
+
+        <div className="flex-grow flex flex-col items-center justify-center">
+          <Container size="lg" mt={4}>
+            <Paper shadow="xs" className="p-20">
+              <div className="mb-6 text-center" id="what-how-section">
+                <Display text="IceBreaker - Break the Ice with Strangers!" />
+              </div>
+
+              <Text className="mt-20">
+                Are you tired of those awkward silences when meeting new people?
+                Do you wish there was a fun and engaging way to connect with
+                strangers? Look no further, because <strong>IceBreaker</strong>{" "}
+                is here to save the day!
+              </Text>
+
+              <Text mt={5}>
+                <strong>What is IceBreaker?</strong>
```

Comment:
This is a nitpick but I think it would be more readable to have larger margins between each section, maybe mt={15} instead of 5? 

I also think the section headers could be slightly larger to make it stand out from the rest of the text more.